Dockerfile

```diff
@@ -34,8 +34,6 @@ ENV BIGCHAINDB_SERVER_BIND 0.0.0.0:9984
 # but maybe our Docker or Docker Compose stuff does?
 # ENV BIGCHAINDB_API_ENDPOINT http://bigchaindb:9984/api/v1
 
-ENTRYPOINT ["bigchaindb", "--dev-start-rethinkdb", "--dev-allow-temp-keypair"]
+ENTRYPOINT ["bigchaindb"]
 
 CMD ["start"]
-
-EXPOSE 8080 9984 28015 29015
```

Comment:
I don't understand why port 9984 isn't exposed. (The other ones are for RethinkDB, which is gone from this container, so that much is clear.)